hyperspy/io_plugins/tiff.py

```diff
@@ -254,6 +252,9 @@ def _read_serie(tiff, serie, filename, force_read_resolution=False,
     else:
         dc = _load_data(*data_args, memmap=memmap, **kwds)
 
+    if _is_streak_hamamatsu(op):
+        op.update({'ImageDescription_Parsed': _get_hamamatsu_streak_description(tiff, op)})
```

Comment:
```suggestion
        op.update({'ImageDescriptionParsed': _get_hamamatsu_streak_description(tiff, op)})
```
Just to avoid mixing CamelCase and snake_case in the same name! ;)